src/style.css

```diff
@@ -0,0 +1,111 @@
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 400;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Avenir-Book, sans-serif;
+  color: rgba(61, 78, 97, 1);
+  font-weight: 300;
+  font-size: 16px;
+}
+
+body {
+  margin: 20px 10px 0;
+}
+
+.form {
+  display: inline-block;
+  width: 100%;
+  margin-bottom: 20px;
+  background: #fff;
+  /* border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px; */
+}
+
+.bigInput {
+  display: inline-block;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  width: 100%;
+  /* height: 70px; */
+  padding: 24px 0;
+  line-height: 20px;
+}
+
+.bigImage {
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-position-y: center;
+  background-position-x: 26px;
+  text-indent: 62px;
+  background-size: 19px;
+}
+
+.smallInput {
+  display: inline-block;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  width: 100%;
+  /* height: 42px; */
+  padding: 11px 0;
+  line-height: 18px;
+  font-size: 14px;
+}
+
+.smallImage {
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-position-y: center;
+  background-position-x: 13px;
+  text-indent: 33px;
+  background-size: 11px;
+}
+
+.form:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.bigInput:focus {
+  background-image: url(images/Search.svg), linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  background-repeat: no-repeat, repeat;
+  border: 1px solid rgba(225, 231, 237, 1);
+  border-radius: 3px;
+  outline: none;
+  font-family: Avenir-Heavy, sans-serif;
+}
+
+.smallInput:focus {
+  background-image: url(images/Search.svg), linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  background-repeat: no-repeat, repeat;
+  border: 1px solid rgba(225, 231, 237, 1);
+  border-radius: 3px;
+  outline: none;
+  font-family: Avenir-Heavy, sans-serif;
+}
+
+.bigInput:focus::placeholder {
+  color: transparent;
+}
+
+.smallInput:focus::placeholder {
+  color: transparent;
+}
```

Comment:
placeholder does not inherit anything. so set the font-size and font-family here explicitly

Reply:
You changed the comment after I got how to do it 🤣

Reply:
sry 😆